libs/designer-ui/src/lib/styles.less

```diff
@@ -68,7 +68,7 @@
 @import './azureResourcePicker/azureResourcePicker.less';
 @import './staticResult/staticResult.less';
 @import './dynamicallyaddedparameter/dynamicallyaddedparameter.less';
-
+@import (reference) './themes.less';
```

Comment:
do we need this?

Reply:
oops sorry forgot to remove that, I was using it before but not anymore